src/Components/WebAssembly/testassets/Wasm.Authentication.Server/Wasm.Authentication.Server.csproj

```diff
@@ -24,6 +25,17 @@
     <Reference Include="Microsoft.EntityFrameworkCore.Tools"  />
   </ItemGroup>
 
+  <Target Name="UpdateIdentityUIStaticWebAssetsConfiguration" AfterTargets="ResolveReferencedProjectsStaticWebAssetsConfiguration">
+    <ItemGroup>
+      <_IdentityUI Include="@(StaticWebAssetProjectConfiguration)" Condition="'%(FileName)' == 'Microsoft.AspNetCore.Identity.UI'" />
+      <_IdentityUI Condition="'%(Identity)' != ''">
+        <AdditionalBuildProperties>@(_IdentityUI->'%(AdditionalBuildProperties)');IdentityDefaultUIFramework=$(IdentityDefaultUIFramework)</AdditionalBuildProperties>
+      </_IdentityUI>
+      <StaticWebAssetProjectConfiguration Remove="@(_IdentityUI)" />
+      <StaticWebAssetProjectConfiguration Include="@(_IdentityUI)" />
+    </ItemGroup>
+  </Target>
+
```

Comment:
This just fixes how we consume identity UI as a class library, which was broken.

Reply:
It would be good to add a comment indicating what all this stuff is for. i.e., in what way it would be broken otherwise, and why it has to be fixed here rather than in whatever added the identity UI files to the project in the first place.

Hopefully you can appreciate that basically nobody in the world would have any idea why we're doing this apart from you :)

Reply:
I'll add a comment for this.